airbyte-integrations/connectors/destination-postgres/src/test-integration/java/io/airbyte/integrations/destination/postgres/PostgresIntegrationTest.java

```diff
@@ -78,17 +79,9 @@ protected JsonNode getFailCheckConfig() {
         ctx -> ctx
             .fetch(String.format("SELECT * FROM %s ORDER BY emitted_at ASC;", streamName))
             .stream()
-            .map(Record::intoMap)
-            .map(r -> r.entrySet().stream().map(e -> {
-              // jooq needs more configuration to handle jsonb natively. coerce it to a string for now and handle
-              // deserializing later.
-              if (e.getValue().getClass().equals(org.jooq.JSONB.class)) {
-                return new AbstractMap.SimpleImmutableEntry<>(e.getKey(), e.getValue().toString());
-              }
-              return e;
-            }).collect(Collectors.toMap(Entry::getKey, Entry::getValue)))
-            .map(r -> (String) r.get(COLUMN_NAME))
+            .map(r -> r.formatJSON(JSON_FORMAT))
```

Comment:
this jsonform  outputs a json object so that we don't need to do all that skullduggery that used to be there.